### ISSUE TITLE
Training loss goes to NaN

### ISSUE THREAD

**Issue by surya501** [carpedm20/BEGAN-tensorflow#11]
I started training with Celeb database, but around 50k iteration, the loss goes to NaN & k_t goes to 1. Have you seen this before.
 10%|█████▎                                               | 49650/500000 [5:29:56<49:13:32,  2.54it/s][49650/500000] Loss_D: 0.112764 Loss_G: 0.053848 measure: 0.1190, k_t: 0.0458
 10%|█████▎                                               | 49700/500000 [5:30:16<49:18:01,  2.54it/s][49700/500000] Loss_D: nan Loss_G: nan measure: nan, k_t: 1.0000

Having said that, the output at 49k is looking good. Below are my last good outputs 
![x_fixed](https://cloud.githubusercontent.com/assets/4899994/25029276/a58f29e6-2070-11e7-8f86-f6e18c79ccb5.png)
![49500_d_real](https://cloud.githubusercontent.com/assets/4899994/25029279/ad5569f6-2070-11e7-8c83-03be951c0372.png)
![49500_g](https://cloud.githubusercontent.com/assets/4899994/25029284/b7296950-2070-11e7-829e-9552984dd0d1.png)
![49500_d_fake](https://cloud.githubusercontent.com/assets/4899994/25029289/bd34f63e-2070-11e7-9463-2b3e594f4805.png)
